react-native/android/build.gradle

```diff
@@ -31,10 +31,17 @@ import org.apache.tools.ant.filters.ReplaceTokens
 // We download various C++ open-source dependencies into downloads.
 // We then copy both the downloaded code and our custom makefiles and headers into third-party-ndk.
 // After that we build native code from src/main/jni with module path pointing at third-party-ndk.
+def npmExecutable = 'npm';
 
-ext.coreVersion = "npm --silent run get-core-version".execute().text.trim()
-ext.syncVersion = "npm --silent run get-sync-version".execute().text.trim()
-def currentVersion = "npm --silent run get-version".execute().text.trim()
+if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+        npmExecutable = 'npm.cmd'
+}
+ext.coreVersion = "$npmExecutable --silent run get-core-version".execute().text.trim()
```

Comment:
@dbhowell The changes in this PR actually make the use of npm platform independent
The build.gradle still uses npm but the npm script is changed like so
https://github.com/realm/realm-js/pull/962/commits/55bc8869048b98bd1bfdccc36db862947bd240de#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R49

You have implemented what @fealebenpae have suggested above for these exact npm scripts, so we can merge your pr with this one.

Still we need to decide what to do with the symlinks in the repo before merging this PR

@fealebenpae @kristiandupont What you guys think?